packages/angular/src/app/components/contact-user-panel/contact-user-panel.component.ts

```diff
@@ -39,33 +40,46 @@ import { Contact } from 'src/app/types/contact';
   styleUrls: ['./contact-user-panel.component.scss'],
   providers: [DataService],
 })
-export class ContactUserPanelComponent implements OnInit, OnChanges, OnDestroy {
+export class ContactUserPanelComponent implements OnInit, OnChanges, AfterViewChecked, OnDestroy {
   @Input() isOpened = false;
 
   @Input() userId: number;
 
   @Output() isOpenedChange = new EventEmitter<boolean>();
 
+  @Output() pinnedChange = new EventEmitter<boolean>();
+
+  private pinEventSubject = new Subject<boolean>();
+
+  private pinEventSubjectDistinct = this.pinEventSubject.pipe(distinctUntilChanged());
```

Comment:
we can move "this.pinEventSubject.pipe(distinctUntilChanged())" to constructor without variable pinEventSubjectDistinct